src/packages/core/modal/token/property-editor-ui-picker-modal.token.ts

```diff
@@ -1,8 +1,6 @@
 import { UmbModalToken } from './modal-token.js';
 
-export interface UmbPropertyEditorUIPickerModalData {
-	submitLabel?: string;
```

Comment:
Should this be marked as deprecated instead of removed? Won't it break the typescript compiler if someone uses it?

Reply:
I'd considered it, I'd made the assumption that no one would be using `UmbPropertyEditorUIPickerModalData` externally yet... but you never know, so yes good point. I'll update with a deprecation. 👍